charts/redhat/redhat/knative-istio-authz/OWNERS

```diff
@@ -0,0 +1,13 @@
+chart:
+  name: knative-istio-authz
+  shortDescription: This is the Red Had Openshift Serverless knative-istio-authz chart
```

Comment:
```suggestion
  shortDescription: Helm chart to onboard a set of authorization-based isolated namespaces to Knative when using Istio
```

Reply:
I guess we have suggested line at chart.yaml Description, this shortDescription is to identify the maintaining team.

Reply:
Ok